xclim/ensembles/_robustness.py

```diff
@@ -116,8 +118,6 @@ def robustness_fractions(  # noqa: C901
     | Negative change | (cf - cpf)         | 1 - pf - (cf -cpf)     | 1 - pf     |
     +-----------------+--------------------+------------------------+------------+
 
-    Notes
-    -----
     Available statistical tests are :
 
     {tests_doc}
```

Comment:
are some tests missing on purpose @aulemahal ? I see ipcc-ar6-c in the example notebook but I can't find it in the code?

Reply:
Woups, the automatic tests doc generation was done on the wrong method. I think I came up with the mechanism before changing the function name.